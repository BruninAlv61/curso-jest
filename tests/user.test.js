const { crearUsuario } = require('../src/user')

test('Debe coincidir con el snapshot', () => {
    const usuario = crearUsuario('Bruno', 30)
    expect(usuario).toMatchSnapshot()
})