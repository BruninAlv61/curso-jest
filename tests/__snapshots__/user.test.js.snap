// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`Debe coincidir con el snapshot 1`] = `
{
  "activo": true,
  "edad": 30,
  "nombre": "Bruno",
  "rol": "usuario libre liberador",
}
`;
